README.md

```diff
@@ -312,6 +271,18 @@ You can leave it enabled, it just won't show anything in the echo area.
 
 #### Code Completion
 
+This section is out of date. Code completion used to work only in lumo
```

Comment:
This is a bit confusing for me, so if there is no completion for a repl I thought we would just not be completing?

Reply:
that's correct. the old way tried to complete if a random library was on the classpath and required. This was often the case with lein but i'm not sure that's a majority of startups these days. So its removed and this section explains how to get it back if desired.

Reply:
Ok cool tnx I did not remember properly then 😁